### ISSUE TITLE
Unread counter incorrect

### ISSUE THREAD

**Issue by trentks** [tutao/tutanota#1316]
**Bug in web app**
Unread counter shows that there are unread emails when there are not.

**Describe the bug**
The unread counter is incorrect---it states there's an unread email when there is not. This issue persists across the mobile interface, desktop interface, and web interface.

Luckily I have only just started using tutanota, some ~2 weeks ago, and so my email count is only a couple hundred or so, so I can easily confirm that none are unread. However, it is also essential that there be a way to filter out unread emails to improve this process---so this would be a new feature request, I might open an issue for this separately.

**To Reproduce**
1. Log in to mail box.
2. See unread email count of 1 in the inbox.
3. Search for unread email, doesn't exist.

**Expected behavior**
I expect to find an unread email.

**Desktop (please complete the following information):**
 - OS: Manjaro 18.0.4, i3 community edition
 - Browser: Firefox Quantum
 - Version: 67.0.4 (64-bit)

**Comment by charlag** [comment]
You should contact support about this (I remember about #1317)

**Comment by lasscap** [comment]
I think this is a bug, as I have the same problem with the 'Inbox' counter.
<img width="868" alt="Screenshot 2019-09-16 at 11 00 17" src="https://user-images.githubusercontent.com/47210775/64945671-59bd6800-d871-11e9-97f9-4b2c2954701e.png">


This error exists for me on macOS Mojave 10.14.6 on Firefox 69 and Safari as well as on Android 8 with the Tutanota App and the web version using Firefox for Android.

**Comment by trentks** [comment]
Hi @lasscap,

I did have to contact support, and they manually reset the counter for me. There's obviously an underlying bug here somewhere that the developers need to fix, but this is a work-around. The question still remains to me whether or not there were an email that I did not actually receive (i.e. was lost, and unreadable), but that incremented the counter. Or, whether the counter may have just incremented twice for a single email (no emails lost).

Anyway, contact support, but I agree that this issue should be reopened as it's a common issue that I've seen quite a few people present with. But, perhaps unfortunately, not-so-common as to be an easily identifiable/replicable bug.

Best of luck getting in contact with support.
